### ISSUE TITLE
Improve Screen Reader and Keyboard Accessibility of Update Manager and Software Manager GUI Applications

### ISSUE THREAD

**Issue by slannon97** [linuxmint/linuxmint#689]
I just installed Mint MATE 22 and the Update Manager and Software Manager applications aren't very accessible with the Orca screen reader. Update Manager is somewhat usable, while the tool to search for and install new software appears to be completely inaccessible. I can get around this by using apt or flatpak commands in the terminal, but these applications should be accessible, particularly for new users that may not be as comfortable using the CLI.
Ideally, it should be possible to tab through all the controls on the screen and have them read properly with Orca. I can currently sort of use the Update Manager by using Orca's review commands, but proper keyboard accessibility would make things significantly easier. All dialog boxes and options should have this improved accessibility, including the preferences dialog in Update Manager.
